quarkus/runtime/src/main/resources/META-INF/keycloak.conf

```diff
@@ -1,13 +1,6 @@
 # Default and non-production grade database vendor
 db=dev-file
 
-# Insecure requests are disabled by default
```

Comment:
No need to set these properties here as their defaults are available from our mappers.